third_party/build/bin/gnutls_build.sh

```diff
@@ -61,6 +66,32 @@ cd ${WORK_DIR}
 wget http://mirrors.dotsrc.org/gcrypt/gnutls/v3.1/gnutls-$PKGVERSION.tar.xz
 tar xf gnutls-$PKGVERSION.tar.xz
 cd gnutls-$PKGVERSION/
+
+patch gl/stdio-impl.h <<'EOF'
```

Comment:
What are the reasons for these patches? Also in `grpc_build.sh`

Reply:
I added some information about the bugs which are fixed by these patches / their upstream fixes in newer versions.